api/nakadi-event-bus-api.yaml

```diff
@@ -1947,16 +1947,16 @@ definitions:
   CursorDistanceResult:
     allOf:
       - $ref: '#/definitions/CursorDistanceQuery'
-    required:
-      - distance
-    properties:
-      distance:
-        type: number
-        format: int64
-        description: |
-          Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
-          are equal.
-
+      - type: Object        
+        properties:
+          distance:
+            type: number
+            format: int64
+            description: |
+              Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
+              are equal.
+        required:
+          - distance
```

Comment:
the ident level is wrong. allOf should contain a list of objects. In this case one is a referense and another is the object defined here. Although the yaml structure is correct(the editor will not complain) the semantical meaning was totally wrong. Any program that try to generate this object will fail or ignore unrecognised field "properties" at the same level as allOff.